components/autocomplete/index.js

```diff
@@ -131,6 +131,13 @@ export class Autocomplete extends Component {
 		this.search = this.search.bind( this );
 		this.handleKeyDown = this.handleKeyDown.bind( this );
 		this.getWordRect = this.getWordRect.bind( this );
+		this.debouncedSearch = debounce( () => {
+			this.search();
+		}, 250 );
```

Comment:
I would try to move the debounce logic into the completer for the users because there will be completers (for example the forward-slash block completer) where it is feasible to load the complete list straight away and it will give a better user experience if they are shown their suggestion list straight away.

Reply:
Ok, I'll try to move it there; I had it there originally and the debounce wasn't working as expected. I agree it would be better to move it there though so I'll give it another pass.